src/sql/base/browser/ui/inputBox/inputBox.ts

```diff
@@ -137,4 +146,19 @@ export class InputBox extends vsInputBox {
 		this.inputForeground = enabled ? this.enabledInputForeground : this.disabledInputForeground;
 		this.inputBorder = enabled ? this.enabledInputBorder : this.disabledInputBorder;
 	}
+
+	public validate(force?: boolean): MessageType | undefined {
```

Comment:
if you merge this in right now, you will need to update the vscode merge pr to add override here, right?

Reply:
Correct, I plan on waiting until that's merged in before getting this in